playground/init.sh

```diff
@@ -1,3 +1,12 @@
 #!/bin/bash
+@Echo off
 
-cd playground_project
+echo Initialising...
+
+cd /tmp
+daml new playground_project
+mkdir -p /root/playground_project
+mv playground_project/* /root/playground_project/
+cd /root
```

Comment:
looks very messed up but if it works I’m happy :)

Reply:
race conditions 🤷‍♂